charts/opentelemetry-demo/values.yaml

```diff
@@ -519,9 +519,17 @@ components:
     env:
       - name: KAFKA_ADVERTISED_LISTENERS
         value: 'PLAINTEXT://{{ include "otel-demo.name" . }}-kafka:9092'
+      - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+        value: 'http://{{ include "otel-demo.name" . }}-otelcol:4317'
```

Comment:
does this need `/v1/traces`?

Reply:
It works without the path suffix. Though I now see the inconsistency across the demo components of using the standard vs telemetry-specific endpoints with and without paths.

Reply:
That is suspicious.  Do we need both env vars are can we use `OTEL_EXPORTER_OTLP_ENDPOINT`?

Reply:
reading the original PR that made this change on the demo side, it seems like the TRACES_ENDPOINT isn't needed and shouldn't of been part of this at all.  I'll make some changes and test just using the non-telemetry specific endpoint and update this PR.

Reply:
Sweet, ping me when you're ready for another review.